docs/release/generate_release_notes.py

```diff
@@ -26,12 +26,25 @@
 """
 import argparse
 import os
+import re
+import sys
 from collections import OrderedDict
+from contextlib import suppress
 from datetime import datetime
+from itertools import chain
+from os.path import abspath
+from pathlib import Path
 from warnings import warn
 
+from git import Repo
 from github import Github
 
+with suppress(ImportError):
+    import requests_cache
+
+    requests_cache.install_cache('github_cache', backend='sqlite')
+    # setup cache for speedup execution and reduce number of requests to GitHub API
+
```

Comment:
when updating labels on GitHub, there is a requirement to remove the cache file. Maybe it should be handled somehow better?